### ISSUE TITLE
Figure 7.3 seems to have a missing tick on the x axis

### ISSUE THREAD

**Issue by he29-net** [hendricius/the-sourdough-framework#390]
Hi,

I'm not quite sure if it is intentional, but Figure 7.3 (dough strength vs. time when leaving it alone) has time ticks of 0, 6, 12, 24, 30 hours etc. – that is, it mostly goes up by 6 hours, but the 18h tick is missing.

Since there are no markings or an explanation in the description, it is not clear whether it is an intentional non-linearity in the time axis, or a typo.

P.S., Great book! I've been baking sourdough on and off for a few years now, but I never perfected the art, since most of the information online was unreliable or conflicting at best – and after a while of experimenting, I reached a "good enough" stage and lost the patience needed to keep tweaking the details. The book is a great resource for finding all the various problems in my baking process, so now I can't wait to ~see~ taste the results after each new attempt. :)

**Comment by cedounet** [comment]
Good point!

I never liked that graph anyway so now a motivation to redo-it :)

thanks

**Comment by cedounet** [comment]
Here we go for the new version... I will fix all graphs before pushing anything though.

![Capture d’écran du 2024-12-29 00-13-28](https://github.com/user-attachments/assets/4d32b96c-e981-4fbc-bcc6-4b07abf06af7)

**Comment by he29-net** [comment]
Looks great (assuming the description is just a placeholder from another figure). :+1: 

Not sure if I should close the issue or if you want to keep it open as a "todo" reminder for the other graphs, so I'll leave that up to you.

**Comment by cedounet** [comment]
> Looks great (assuming the description is just a placeholder from another figure). 👍
> 
Yes… I only compile one figure not the full book… takes long enough already… my previous MWE likely already had that figure and forgot to change the caption.  Sorry about that.

> Not sure if I should close the issue or if you want to keep it open as a "todo" reminder for the other graphs, so I'll leave that up to you.

Leave it with me… I did not merge it in yet, idea really is to have same font/colours as the main text and searchable/copy pastable text in those graphs.

I do have questions on when exactly examdaple stop and decay start?

thanks.

**Comment by hendricius** [comment]
Looking great. It's not exactly defined unfortunately, it's more a schematic visualisation of what's happening. The key takeaway should be that the sourdough based dough decays faster than the yeast based dough. So it's looking good to me 👍

**Comment by cedounet** [comment]
https://github.com/hendricius/the-sourdough-framework/pull/412
